io.github.Dretch.MonomerFlatpakExample.json

```diff
@@ -0,0 +1,3423 @@
+{
+  "app-id": "io.github.Dretch.MonomerFlatpakExample",
+  "cleanup": [
+    "/lib/x86_64-linux-ghc-9.2.5",
+    "/share/doc/x86_64-linux-ghc-9.2.5",
+    "/bin/alex",
+    "/bin/c2hs",
+    "/bin/dev-test-app",
+    "/bin/happy",
+    "/share/*-ghc-*/alex-*",
+    "/share/*-ghc-*/c2hs-*",
+    "/share/*-ghc-*/happy-*",
+    "/share/*-ghc-*/sdl2-*"
+  ],
+  "command": "monomer-flatpak-example",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "builddir": false,
+      "buildsystem": "simple",
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.2.5",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.2.5",
+        "/bin/ghci",
+        "/bin/ghci-9.2.5",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.2.5",
+        "/bin/runghc",
+        "/bin/runghc-9.2.5",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.2.5",
+        "/share/man",
+        "/share/doc/ghc-9.2.5"
+      ],
+      "name": "ghc-9.2.5-x86_64",
+      "only-arches": [
+        "x86_64"
+      ],
+      "sources": [
+        {
+          "sha256": "2d115b7258751f0e4481e35b5953ca3c7870e8ec9ce68f1d32fc014ddc29b2a5",
+          "type": "archive",
+          "url": "https://downloads.haskell.org/ghc/9.2.5/ghc-9.2.5-x86_64-deb9-linux.tar.xz"
+        }
+      ]
+    },
+    "extra-libraries/glu.json",
+    "extra-libraries/glew.json",
```

Comment:
you should use glu and glew from shared-modules

https://github.com/flathub/shared-modules/

Reply:
glu works fine here.